totalRP3/modules/register/characters/register_ui_characteristics.xml

```diff
@@ -237,6 +237,7 @@
 					<Anchor point="TOPLEFT" relativeKey="$parent.Bar" x="-16"/>
 					<Anchor point="BOTTOMRIGHT" relativeKey="$parent.Bar" x="16"/>
 				</Anchors>
+				<HitRectInsets left="12" right="12" top="0" bottom="0" />

```

Comment:
Just to clarify, we don't use the 16px offset for the inset here because otherwise the extreme edges of the slider wouldn't be clickable to set the bar to its min/max value. 12px seems to be a good compromise between not cutting either widget off too much.